SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/DeploymentLauncher/DeploymentLauncher.cs

```diff
@@ -174,9 +174,12 @@ private static int CreateDeployment(string[] args, bool useChinaPlatform)
 
                 if (launchSimPlayerDeployment)
                 {
+                    // we are using the main deployment snapshot also for the sim player deployment, because we only need to specify a deployment
```

Comment:
```suggestion
                    // we are using the main deployment snapshot also for the sim player deployment, because we only need to specify a snapshot
```